### ISSUE TITLE
DatePicker onBlur of undefined

### ISSUE THREAD

**Issue by dehrax** [erikras/redux-form-material-ui#82]
I am trying to implement the DatePicker, but I seem to get a weird error

`import { DatePicker } from 'redux-form-material-ui';`

This is the field with the DatePicker component.

```javascript
<Field
name="startDate"
component={DatePicker}
floatingLabelText={ <Msg {...messages.startDate}/> } // react-intl formatted message
onChange={(value) => {
	console.log(value);
}}
format={null}
/>
```

I am not trying to use onBlur anywhere, which is why it is weird that I get this error:
```javascript
Uncaught TypeError: Cannot read property 'onBlur' of undefined
    at eval (eval at ./node_modules/redux-form-material-ui/lib/DatePicker.js (bundle.js:15788), <anonymous>:27:26)
    at InputComponent.render (eval at ./node_modules/redux-form-material-ui/lib/createComponent.js (bundle.js:15844), <anonymous>:46:76)
    at eval (eval at ./node_modules/react-dom/lib/ReactCompositeComponent.js (bundle.js:6379), <anonymous>:796:21)
    at measureLifeCyclePerf (eval at ./node_modules/react-dom/lib/ReactCompositeComponent.js (bundle.js:6379), <anonymous>:75:12)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (eval at ./node_modules/react-dom/lib/ReactCompositeComponent.js (bundle.js:6379), <anonymous>:795:25)
    at ReactCompositeComponentWrapper._renderValidatedComponent (eval at ./node_modules/react-dom/lib/ReactCompositeComponent.js (bundle.js:6379), <anonymous>:822:32)
    at ReactCompositeComponentWrapper.performInitialMount (eval at ./node_modules/react-dom/lib/ReactCompositeComponent.js (bundle.js:6379), <anonymous>:362:30)
    at ReactCompositeComponentWrapper.mountComponent (eval at ./node_modules/react-dom/lib/ReactCompositeComponent.js (bundle.js:6379), <anonymous>:258:21)
    at Object.mountComponent (eval at ./node_modules/react-dom/lib/ReactReconciler.js (bundle.js:6731), <anonymous>:46:35)
    at ReactDOMComponent.mountChildren (eval at ./node_modules/react-dom/lib/ReactMultiChild.js (bundle.js:6675), <anonymous>:238:44)
```

**Comment by erikras** [comment]
What RF and RFMUI versions?

**Comment by Giboork** [comment]
We are currently using 4.1.2, and experiencing the very same thing.

**Comment by Giboork** [comment]
The same goes for TimePicker.

**Comment by she-dev** [comment]
I'm getting a similar error: Cannot read property 'onChange' of undefined.

_ref is defined contains onChange, it seems that _ref.input isn't:
![](http://i.imgur.com/qTa9DAK.png)

**Comment by lxm7** [comment]
Did anyone find a work around for this?

**Comment by kevinverelst** [comment]
Having the same issue with the datePicker

**Comment by mihirsoni** [comment]
It seems working with latest version MUI 0.18 and RFMUI 4.3 

You can take a look at example here:

https://github.com/erikras/redux-form-material-ui/blob/4336ce2cec402d84e37bbd763b0f8009d721144d/example/src/Form.js#L112-L118

Feel free to re-open if issue still persist.

**Comment by vladimirivanoviliev** [comment]
I'm not sure why @mihirsoni closed the issue as the MUI 0.18 is NOT the latest version and with latest 0.19x is still reproducible and blocking. What is your suggestion - to downgrade to v.18x ?

**Comment by matteodanelli** [comment]
The issue persist with MUI 0.19.4, the latest before the 1.0.
RFMUI updated to 4.3.

Cleaned all packages and re-installed but nothing changed.
